docs/markdown/Writing Plugins/rules-api/rules-api-goal-rules.md

```diff
@@ -108,13 +108,13 @@ async def hello_world(
 
 ### `LineOriented` mixin (optional)
 
-If your goal's purpose is to emit output—and that output is naturally split by new lines—it may be helpful to use the mixin `LineOriented`. This subclasses `Outputting`, so will register both the options `--output-file` and `--sep`, which allows the user to change the separator to not be `\n`.
+If your goal's purpose is to emit output -- and that output is naturally split by new lines -- it may be helpful to use the mixin `LineOriented`. This subclasses `Outputting`, so will register both the options `--output-file` and `--sep`, which allows the user to change the separator to not be `\n`.
```

Comment:
I am not sure if we shall replace all readme.com em dashes (`—`) with two single dashes (`--`) and rely instead of the docs generation tool to convert double dashes into an em dash, if desired? :/

Reply:
It's probably better to not have em dashes in the source, where they are likely to get overwritten as a regular dash.

Reply:
Right. As we are still with readme.com, can we confirm double dash (`--`) will get converted to an em dash during the rendering? If so, I am happy to raise another PR replacing all em dashes with double dashes.

Reply:
I have not looked into this, but if that's the case then it's a good idea to use double dash in the source, yes.